src/openapi/streaming/connection/transport/signalr-core-transport.js

```diff
@@ -24,15 +24,46 @@ function handleLog(level, message) {
     log.warn(LOG_AREA, message);
 }
 
-function buildConnection({ baseUrl, contextId, accessTokenFactory, protocol }) {
+function getRetryPolicy(transport) {
+    return {
+        nextRetryDelayInMilliseconds(retryContext) {
+            if (transport.authExpiry < Date.now()) {
+                log.warn(LOG_AREA, 'Token expired while trying to reconnect');

```

Comment:
its fine that this is a warn to start with. later on we might take it to an info, since it is a normal, ok scenario